### PR TITLE
fix: Optimize monthly stat API

### PR DESCRIPTION
**Pull request by fregataa**
So far, monthly stat API do iteration for (2880 * num_kernels) times constantly and fetch Kernel stats from Redis for every iteration.
This PR reduce the iteration and the number of Redis fetch.

# Before
Waiting for server response 4.77 s
* Not "milisecond", it is "second"
image

# After
Waiting for server response 140.08 ms
image

### CODE REVIEW

**Review by achimnol — APPROVED** [review]
LGTM for me, but need to confirm that the result is same to the previous implementation. @adrysn

File: src/ai/backend/manager/api/resource.py [hunk 1]
```diff
@@ -605,74 +613,86 @@ async def get_time_binned_monthly_stats(request: web.Request, user_uuid=None):
...
-            if raw_stat:
-                last_stat = msgpack.unpackb(raw_stat)
-                io_read_bytes += int(nmget(last_stat, "io_read.current", 0))
-                io_write_bytes += int(nmget(last_stat, "io_write.current", 0))
-                disk_used += int(nmget(last_stat, "io_scratch_size/stats.max", 0, "/"))
-        stat = {
-            "date": ts,
+    time_series_list: list[dict[str, Any]] = [
+        {
+            "date": idx * time_window + now_ts,
```
Comment (adrysn) [inline src/ai/backend/manager/api/resource.py]: `date` calculation is incorrect. The `date` is starting from `now_ts` in this logic, but this should start from one month ago. Please refer to the previous logic, where `ts` starts from 30 days ago: https://github.com/lablup/backend.ai/pull/1214/files?w=1#diff-51adf4e1db91582bd045aea8a34e9565383e2c437c8cf7aa9eab64aebbdf9b3cL643

**Review by adrysn — CHANGES_REQUESTED** [review]
The value of time in the time series data is incorrect.

File: src/ai/backend/manager/api/resource.py (hunk 1 above)
Comment (fregataa) [reply src/ai/backend/manager/api/resource.py]: fix this part. thank you!

**Review by adrysn — APPROVED** [review]
LGTM.

**Pull request merged**